aggregator_core/src/datastore.rs

```diff
@@ -1112,20 +1112,28 @@ impl<C: Clock> Transaction<'_, C> {
     {
         let time = Time::from_naive_date_time(&row.get("client_timestamp"));
 
-        let encoded_extensions: Vec<u8> = row.get("extensions");
+        let encoded_extensions: Vec<u8> = row
+            .get::<_, Option<_>>("extensions")
+            .ok_or_else(|| Error::IncompleteReport)?;
```

Comment:
But I want to panic if the value can't be decoded, as this represents programmer error: either "asking for a column I didn't actually read" or "trying to read into an inconsistent type". (The specific programmer error before this PR was "trying to decode a `NULL`able column into a `Vec<u8>` rather than an `Option<Vec<u8>>`".) Generally, `Row::get` should be used with an appropriate output type, as long as the output type is known.

Reply:
Yeah, that makes sense. I was concerned about the case where there is a value in the `extensions` column but it's malformed, but you're correct that this case will panic and not get flattened into `IncompleteReport`. This is fine as-is.